### PR TITLE
raidboss: DSR P6 & P7 some triggers

### PR DESCRIPTION
**Pull request by Azruine**
closed #4527 and reopened

### CODE REVIEW

File: ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts [hunk 1]
```diff
@@ -1652,6 +1652,79 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'Spreading Flame',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:1656]:
```suggestion
      id: 'DSR Spreading Flame',
```

File: ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts [hunk 2]
```diff
@@ -1652,6 +1652,79 @@ const triggerSet: TriggerSet<Data> = {
...
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Spread',
+          ko: '산개징 대상자',
+        }
+      }
+    },
+    {
+      id: 'Entangled Flame',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:1671]:
```suggestion
      id: 'DSR Entangled Flame',
```

**Review by quisquous — COMMENTED** [review]
You're also missing a bunch of commas and semicolons in lint: <https://github.com/quisquous/cactbot/pull/4533/files>

If you have npm installed, and do an `npm install` and an `npm run lintfix` it will fix them automatically for you.

**Comment by Azruine** [comment]
> You're also missing a bunch of commas and semicolons in lint: https://github.com/quisquous/cactbot/pull/4533/files
> 
> If you have npm installed, and do an `npm install` and an `npm run lintfix` it will fix them automatically for you.

ty, lint fix done :)

**Pull request merged**